src/components/pacbio/PacbioSampleMetadataEdit.vue

```diff
@@ -93,9 +100,20 @@ export default {
     },
     ...mapActions(['updateRequest']),
     show() {
-      this.$refs['modal'].show()
+      this.showModal = true
+      /**This need to be removed when custom_enable_modal feature flag is removed */
+      if ('b-modal' in this.$refs['modal'].$refs) {
```

Comment:
This looks to be the same as what is going on in src/components/pacbio/PacbioRunWellEdit.vue. Maybe if we move them to the same place it will be easier to remove when the feature flag is removed?

Reply:
Done 👍

Reply:
Did you mean combining all of this similar statements _this.$refs['enzymeModal'].$refs['b-modal'].show()_ to one place. I am afraid that will require quite a bit of refactoring , with each checking it's own ref in it's own instance (this) context